app/assets/javascripts/views/holiday.js

```diff
@@ -0,0 +1,108 @@
+App.Views.Holiday = Backbone.View.extend({
+  tagName: 'li',
+  className: 'list-group-item',
+  template: JST['templates/holiday'],
+
+  events: function() {
+    var role = App.currentUserRoles.highestPrivilege(),
+        result = {};
+
+    switch (role) {
+      case 'manager':
+        result = {
+          'click button[name=delete]': 'onDelete',
+          'dblclick .view': 'enterEditMode',
+          'click button[name=update]': 'onUpdate',
+          'click button[name=cancel]': 'onCancel'
+        };
+        break;
+
+      default:
+        result = {};
+    }
+
+    return result;
+  },
+
+  initialize: function(options) {
+    this.model = options.model;
+    this.attributes = {
+      role: App.currentUserRoles.highestPrivilege()
+    };
+  },
+
+  render: function() {
+    this.populateAttributesWithData();
+
+    this.$el.html(this.template(this.attributes));
+
+    return this;
+  },
+
+  populateAttributesWithData: function() {
+    this.attributes.description = this.model.get('description');
+    this.attributes.from = this.model.get('start');
+    this.attributes.to = moment(this.model.get('start')).add(this.model.get('duration')-1, 'day').format('YYYY-MM-DD');
+    this.attributes.date = moment(this.model.get('start')).format('MMMM D, dddd');
+    this.attributes.duration = moment.duration(this.model.get('duration'), 'days').humanize();
+  },
+
+  enterEditMode: function(event) {
+    this.$el.addClass('editing');
+    App.Helpers.assignDatePicker($('.input-daterange'));
+  },
+
+  exitEditMode: function() {
+    this.$el.removeClass('editing');
+  },
+
+  onCancel: function(event) {
+    this.exitEditMode();
+  },
+
+  onUpdate: function(event) {
+    this.close();
+  },
+
+  fetchFormValues: function() {
+    var _description = this.$('input[name=description]').val().trim(),
+        _from = this.$('input[name=from]').val().trim(),
+        _to   = this.$('input[name=to]').val().trim(),
+        duration = moment.duration(moment(_to).diff(moment(_from))).days()+1,
```

Comment:
I think this chain `moment.duration(moment(param1)).diff(moment(param2)).days()+1` should move to `function(param1, param2)`. Because you use this structure many times.